### PR TITLE
Make cleanup easier.

### PR DESCRIPTION
**Pull request by vishh**

### CODE REVIEW

**Comment by pwittrock** [comment]
@vishh Mind rebasing and then I can LGTM

**Comment by bgrant0607** [comment]
@foxish @apelisse @pwittrock @Kashomon Example of a doc PR that could benefit from the bot.

**Comment by Kashomon** [comment]
@bgrant0607 -- can you elaborate? What features of the bot do you think apply here?

**Comment by apelisse** [comment]
This didn't get attention until ... now?

**Comment by bgrant0607** [comment]
@apelisse @Kashomon Yes, look how long this PR just sat around with no activity -- from March until ~week ago, and a week since then. There is no assigned reviewer. There is no `needs-rebase` label.

**Comment by bgrant0607** [comment]
@luxas PTAL

**Comment by kelseyhightower** [comment]
@vishh  Please rebase this and I'll get it over the finish line.

File: docs/getting-started-guides/docker.md [hunk 1]
```diff
@@ -145,14 +145,14 @@ See [here](/docs/getting-started-guides/docker-multinode/deployDNS/) for instruc
...
 
-1. Delete all the containers including the kubelet:
+* Delete all the containers including the kubelet:
 
-Many of these containers run under the management of the `kubelet` binary, which attempts to keep containers running, even if they fail.
-So, in order to turn down the cluster, you need to first kill the kubelet container, and then any other containers.
-
-You may use `docker kill $(docker ps -aq)`, note this removes _all_ containers running under Docker, so use with caution.
+```shell
+docker rm -f kubelet
```
Comment (luxas) [inline docs/getting-started-guides/docker.md]: The kubelet docker image isn't named kubelet anymore

Comment (luxas) [inline docs/getting-started-guides/docker.md]: The best way is probably to name the kubelet container `k8s_$(random-3char-hash)`, and then it'll be catched by the line below

Comment (vishh) [inline docs/getting-started-guides/docker.md]: I added the `kubelet` container name back. It is the most reliable way to identify kubelet.

**Comment by vishh** [comment]
Rebased and comments addressed. PTAL.

**Comment by luxas** [comment]
LGTM

**Comment by vishh** [comment]
Rebased again.

**Comment by vishh** [comment]
@luxas @kelseyhightower PTAL

**Pull request merged**